src/components/Dialog/Dialog.js

```diff
@@ -117,27 +117,17 @@ const propTypes = {
   classes: PropTypes.object,
 };
 
-const defaultProps = {
-  title: null,
-  body: null,
-  // footer: null,
-  isModal: false,
-  className: null,
-  onClose: null,
-  classes: {},
-};
-
 function Dialog(props) {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const {
-    body,
-    title,
+    body = null,
+    title = null,
     name,
-    isModal,
-    className,
-    onClose,
-    classes: classesProp,
+    isModal = false,
+    className = null,
+    onClose = null,
+    classes: classesProp = {},
```

Comment:
This make "classes" always a new value everytime, so it will re render the component that use it

Reply:
ok, I changed it to: 
const classesProp = classes || {};
inside the component